spanner/integration_test.go

```diff
@@ -1647,7 +1644,6 @@ func TestIntegration_BasicTypes(t *testing.T) {
 		{col: "String", val: "foo", want: NullString{"foo", true}},
 		{col: "String", val: NullString{"bar", true}, want: "bar"},
 		{col: "String", val: NullString{"bar", false}, want: NullString{"", false}},
-		{col: "String", val: nil, want: NullString{}},
```

Comment:
Removed as it was a duplicate (the same test is on line 1737). This also applies to a couple of other tests below.